src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.DebugVerifier.cs

```diff
@@ -57,6 +56,12 @@ protected override BoundExpression VisitExpressionWithoutStackGuard(BoundExpress
 
             public override BoundNode Visit(BoundNode node)
             {
+                // Ensure that we always have a checkpoint for every BoundExpression in the map
```

Comment:
>BoundExpression [](start = 69, length = 15)

Is the comment accurate? I do not see anything specific for BoundExpression nodes.